app/static/js/bonnerManagement.js

```diff
@@ -76,6 +76,9 @@ $(document).ready(function(){
         } else {
             fileName = `Bonner Spreadsheet, ${Number(startingYear) - Number(noOfYears)} - ${startingYear}`;
         }
+        // const fileName = noOfYears === "all" 
+        //     ? "Bonner Spreadsheet, All Cohorts" 
+        //     : `Bonner Spreadsheet, ${Number(startingYear) - Number(noOfYears) + 1} - ${Number(startingYear) + 1}`
```

Comment:
What's up with this commented out section? It seems like it makes more sense than the if statement above it. Was it causing an error having fileName redefined?